Configuration/StandardSequences/python/Validation_cff.py

```diff
@@ -103,3 +103,23 @@
 from Configuration.Eras.Modifier_fastSim_cff import fastSim
 fastSim.toReplaceWith(validation_prod,_validation_prod_fastsim)
 
+from Validation.EcalClusters.ecalClustersValidationSequence_cff import ecalClustersValidationSequence
```

Comment:
@tanmaymudholkar @jfernan2 I repeat here my comment on the 10_6_X backport https://github.com/cms-sw/cmssw/pull/27483#pullrequestreview-265681038 : this does not seem to me the best place where to insert this definition, that should better go in the ECAL validation fragment.

Reply:
@fabiocos @jfernan2 Indeed. Sequence definition moved to better location in the latest commit.